components/odh-notebook-controller/README.md

```diff
@@ -78,6 +78,21 @@ Run the following command to execute them:
 make test
 ```
 
+Useful options for running tests (that are already included in the above `make` command) are
+
+| Option                     | Description                                                                                   |
+|----------------------------|-----------------------------------------------------------------------------------------------|
+| KUBEBUILDER_ASSETS=        | Environment variable that specifies path to where Kubebuilder has downloaded envtest binaries |
```

Comment:
Is the `=` character intended here?
```suggestion
| KUBEBUILDER_ASSETS        | Environment variable that specifies path to where Kubebuilder has downloaded envtest binaries |
```

Reply:
Wanted to highlight it's a var... but yeah, it's not really helpful and it looks ugly.